install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -514,4 +518,4 @@ ALTER TABLE `PREFIX_web_browser` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb
 ALTER TABLE `PREFIX_zone` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci;
 ALTER TABLE `PREFIX_zone_shop` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci;
 
-/* PHP:ps_1770_preset_tab_enabled(); */
+/* PHP:ps_1770_preset_tab_enabled(); */;
```

Comment:
No need to add ; in comments
```suggestion
/* PHP:ps_1770_preset_tab_enabled(); */
```

Reply:
No actually the ending `;` is CRUCIAL because the regexp used to extract the php function name is based on it Without this ending `;` the php call AND the following SQL call don't work because they are mixed up (took me a while to understand that)

I you look at the previous SQL upgrade files you'll see that every PHP comments have this at the end

Reply:
It's maybe time to fix the regex :joy:  (Of course in a new pull request)